app/scripts/controllers/createPersistentVolumeClaim.js

```diff
@@ -34,6 +35,21 @@ angular.module('openshiftConsole')
       }
     ];
 
+    var pvcTemplate = {
+      kind: "PersistentVolumeClaim",
+      apiVersion: "v1",
+      metadata: {
+        name: $scope.claim.name,
```

Comment:
`$scope.claim.name` is not set at this point.

Reply:
Oop, thx, fixed & tested.